rsts/user/getting_started/create_first.rst

```diff
@@ -14,18 +14,15 @@ You can save some effort by cloning the ``flytesnacks`` repo, and re-initializin
   git init
   cd python
```

Comment:
do we need to clone? and this is wrong if so

what about 
Step 1: lets create a new folder
`` mkdir my_example``
2. Create this structure
    my_example/workflows
3.  pip install flytekit - `pip install flytekit==0.16.0b1`
4. now edit a file named
`my_example/workflows/x.py`

AS a follow up @wild-endeavor maybe we can have a dummy repo in flyteorg, flytekit starter template project?
then we can git-clone.

Then in the next section, `want to get this code running on a flyte deployment?`
1. Setup a sandbox deployment on your local machine
2. ,,,
a. Add a dockerfile
b. Add a config
c. run a commnad
d. run from ui

Reply:
yeah, I like this structure.  We should eventually do a template of some kind yes.  I'll look into a couple options early next week and pick one.

Reply:
yes i'd like to have a working/copy-pastable example for the local sandbox story